requirements.txt

```diff
@@ -1,4 +1,4 @@
-fonttools[ufo]
+fonttools[ufo,lxml,woff,unicode,interpolatable]
```

Comment:
Could you move the dependency updates to a new pr please?

May be worth discussing these first.